src/scenario.py

```diff
@@ -576,10 +584,10 @@ def run(self, strategy_name, options):
             for name, values in loads.items():
                 ax.plot(xlabels, values, label=name)
             # draw schedule
-            for gcID, schedule in gcPowerSchedule.items():
```

Comment:
schedule isn't plotted anymore? Why not both? Maybe draw a dashed line for window?